packages/date-picker/src/panel/date.vue

```diff
@@ -492,7 +551,7 @@
         return this.defaultValue ? new Date(this.defaultValue) : new Date();
       },
 
-      checkDateWithinRange(date) {
+      checkTimeWithinRange(date) {
```

Comment:
Rename to make it more clear/accurate (its actually checking the time component of the date)